pkg/controller/garbagecollector/graph.go

```diff
@@ -67,10 +72,10 @@ func (n *node) isBeingDeleted() bool {
 	return n.beingDeleted
 }
 
-func (n *node) setDeletingDependents(v bool) {
+func (n *node) setDeletingDependentsToTrue() {
```

Comment:
I would still prefer "beginDeletingDependents" or just plain "setDeletingDependents"... the "ToTrue" at the end is a bit awkward. I think just plain "set" or "mark" isn't too bad on a one-way field like this.